zhaquirks/linxura/__init__.py

```diff
@@ -0,0 +1,57 @@
+"""Linxura button device."""
+
+from zigpy.quirks import CustomCluster
+import zigpy.zcl.clusters.security
+
+from zhaquirks.const import (
+    BUTTON,
+    BUTTON_1,
+    BUTTON_2,
+    BUTTON_3,
+    BUTTON_4,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_ID,
+    COMMAND_PRESS,
+    PRESS_TYPE,
+    ZHA_SEND_EVENT,
+)
+
+LINXURA = "Linxura"
+STATUS_REPORT = 2
+
+
+PRESS_TYPES = {
+    1: COMMAND_PRESS,
+    2: COMMAND_DOUBLE,
+    3: COMMAND_HOLD,
+}
+
+
+class LinxuraIASCluster(CustomCluster, zigpy.zcl.clusters.security.IasZone):
```

Comment:
This all seems to be relevant to only the one button quirk for now, so I'd move everything from this file into `button.py`. This follows what we do with other quirks.
Only if another quirk (file) also needs to access it, should it be moved here.
This file should just contain the docstring from the top then.